src/widgets/forms.js

```diff
@@ -1183,7 +1197,7 @@ forms.makeDescription = function (
   }
   function saveChange (_e) {
     submit.disabled = true
-    submit.setAttribute('style', 'visibility: hidden; float: right;') // Keep UI clean
+    submit.setAttribute('style', ' : hidden; float: right;') // Keep UI clean
```

Comment:
**question**: Is this intentional? I wonder why `' : hidden;` is left without a preceding css property

Reply:
Fixed in latest commit